examples/jms-service-consume-message/jms_service_consume_message.md

```diff
@@ -0,0 +1,18 @@
+# JMS service - Consume message
```

Comment:
```suggestion
# JMS service - Consume messages
```
The BBE above had a similar topic. Would be better if we can change the title to not sound the similar.